src/upload/upload.controller.spec.ts

```diff
@@ -0,0 +1,18 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import { UploadController } from './upload.controller';
+
+describe('UploadController', () => {
+  let controller: UploadController;
+
+  beforeEach(async () => {
```

Comment:
if i have the right mind mr mostafavi told us to use beforeAll instead of beforeEach so that the test takes less time to work!